unique_toolkit/unique_toolkit/embedding/utils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import tiktoken
+
+
+def count_tokens(text: str, encoding_model="cl100k_base") -> int:
```

Comment:
Token count does not have to be related to embedding. So I am not sure if it is here at the right place or better at content